synapse/lib/layer.py

```diff
@@ -61,14 +61,10 @@ def allowed(self, perm):
         async for item in self.layr.iterUnivRows(univ):
             yield item
 
-    async def stor(self, sops):
+    async def stor(self, sops, prov=None, splices=None):
```

Comment:
Since the LayerApi is used for a remote layer; and the case where the remote layer *may* be a write layer, why are we not storing splices/prov data there?